### PR TITLE
Update selenium, chromedriver and firefox

### PR DESCRIPTION
**Pull request by vrutkovs**
Make sure #1 is merged first
- Make versions parametrized
- Update selenium, chromedriver and firefox
- Rework startup procedure to be more stable

### CODE REVIEW

File: xstartup.sh [hunk 1]
```diff
@@ -8,12 +8,13 @@ echo $PASS | passwd --stdin root
 /usr/sbin/sshd
 
 export DISPLAY=:99
-Xvfb :99 -shmem -screen 0 1280x1024x16 &
-x11vnc -display :99 -N -shared -forever &
 export PATH="/usr/bin:/root/firefox:/root/chrome-driver:$PATH"
 
-# Start the window manager
+Xvfb $DISPLAY -shmem -screen 0 '1280x1024x16' &
+sleep 5
```
Comment (psav) [inline xstartup.sh:20]: Did it cause problems without this?

File: Dockerfile [hunk 2]
```diff
@@ -1,21 +1,35 @@
 # sections ordered from least likely to change to most
 FROM cfmeqe/sel_base:latest
+
+ENV CHROME_DRIVER_VERSION 2.21
```
Comment (psav) [inline Dockerfile:4]: Like this a lot! I presume this sets a default? We should see what the current FF supported version is. Both the chrome driver and selenium bump seemed to work with the framework OK?

Comment (vrutkovs) [inline Dockerfile:4]: > I presume this sets a default? 

Yep, it allows us to do custom containers and make version bumps clear

> Both the chrome driver and selenium bump seemed to work with the framework OK?

Yep, I tried smoketest only though.

File: xstartup.sh (hunk 1 above)
Comment (vrutkovs) [inline xstartup.sh:20]: Yep, x11vnc failed to start locally as Xvfb didn't finish init phase. Not sure if it happens on prod, but it crashed ~50% of local containers

**Pull request merged**